WordPress/gutenberg
# Add justification to block toolbar in addition to sidebar

## What?


Added justification options to the block toolbar for "type constrained" blocks.

Fixes: #60838 

## Why?


The justification settings for other blocks, such as text blocks, are found in the toolbar. Many users overlook the settings when they are only in the sidebar, expecting them to be in the toolbar. Adding the justification options to the toolbar improves consistency and user experience.

## How?


Added Justification `toolBarControls` of the constrained layout type blocks.

## Testing Instructions





- Add any "type constrained" block, such as Group or Content.
- Check the toolbar to see if the justification option is available there.
- Verify that the justification options are functional and behave as expected.

### Testing Instructions for Keyboard


## Screenshots or screencast 

<img width="1470" alt="Screenshot 2024-06-27 at 5 54 32 PM" src="https://github.com/WordPress/gutenberg/assets/78657883/2a364f05-8bcd-4bd9-96c8-e50c9085487c">
<img width="1466" alt="Screenshot 2024-06-27 at 5 54 58 PM" src="https://github.com/WordPress/gutenberg/assets/78657883/f2674202-ed36-40b0-9753-410bb2716697">
